python/tests/unit/arcticdb/version_store/test_parallel.py

```diff
@@ -142,8 +142,8 @@ def test_write_parallel_sort_merge(basic_arctic_library, prune_previous_versions
     dataframes = []
     df = pd.DataFrame()
 
+    cols = random.sample(columns, 4)
```

Comment:
Assume this change is necessary because `basic_arctic_library` has static schema? In which case, just generate 4 random column names instead of generating 8 and sampling 4